@here/harp-datasource-protocol/lib/operators/ColorOperators.ts

```diff
@@ -7,27 +7,46 @@
 import * as THREE from "three";
 
 import { CallExpr } from "../Expr";
-
 import { ExprEvaluatorContext, OperatorDescriptorMap } from "../ExprEvaluator";
 
 const tmpColor = new THREE.Color();
 const operators = {
+    rgba: {
+        call: (context: ExprEvaluatorContext, call: CallExpr) => {
+            const r = context.evaluate(call.args[0]);
```

Comment:
please add a paragraph to StyleExpressions.md describing this operator

Reply:
Done.